CHANGELOG.md

```diff
@@ -0,0 +1,13 @@
+# Change Log
+
+All notable changes to this project will be documented in this file.
+
+## Unreleased - as of Sprint 53 end 2017-01-30
+
+### Added
+- Refactor `#endpoint` [(#64)](https://github.com/ansible/ansible_tower_client_ruby/pull/64)
+- Expose playbooks off of projects [(#62)](https://github.com/ansible/ansible_tower_client_ruby/pull/62)
+- Add update and delete methods to all exposed classes [(#61)](https://github.com/ansible/ansible_tower_client_ruby/pull/61)
```

Comment:
I think `update_attributes`, `save` and `destroy` may be better here since those are the official method names.

Officially I include the `!` version and the `boolean` of each of the above.  Not sure how relevant that is here.